ref/general/hardening-intro.adoc

```diff
@@ -13,7 +13,7 @@
 :page-type: general
 = Hardening Open Liberty servers in production
 
-Hardening is a general term that is used to describe the measures that are taken to enhance your system security against potential threats and risks. It is important to harden your Open Liberty server to reduce vulnerabilities and prevent against different types of intrusions that can occur. Security intrusions and vulnerabilities can be broken down into different groups:
+Hardening is a general term that is used to describe the measures that are taken to enhance your system security against potential threats and risks in production. It is important to harden your Open Liberty server to reduce vulnerabilities and prevent against different types of intrusions that can occur. Different types of security intrusions can threaten different parts of your system, and there are best practices that you can employ to make each part less vulnerable. Security intrusions and vulnerabilities can be broken down into different groups:
```

Comment:
`It is important to harden your Open Liberty server to reduce vulnerabilities and prevent against different types of intrusions that can occur.`

could be more concise-

`It is important to harden your Open Liberty server to reduce vulnerabilities and prevent security intrusions.`

Reply:
Updated